extensions/env-injector/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "env-exporter"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+http-body-util = "0.1.2"
+hyper = {git = "https://github.com/hyperium/hyper", branch = "master"}
```

Comment:
How reliably stable are these branches? Especially `hyperlocal`, since it's pre-1.0.

I think I'd prefer to have these use frozen versions and rely on renovate updating these for snapcraft. (For `hyper` "any 1.x release >= some number" plus relying on renovate for major releases would also be fine IMO.)

Reply:
Thanks for the suggestion. This was during the development/hacking phase, and I ended up forgetting to use a fixed tag/commit hash. I agree with your point, and I have now changed it to a fixed tag.